OptimizelySDKCore/OptimizelySDKCoreTests/OPTLYDecisionServiceTest.m

```diff
@@ -471,7 +471,7 @@ - (void)testSetForcedVariationCalledOnInvalidVariationKey2
 - (void)testSetForcedVariationCalledOnInvalidUserId
 {
     NSString *invalidUserId = @"";
-    XCTAssertFalse([self.optimizely setForcedVariation:kExperimentNotRunningKey
+    XCTAssertTrue([self.optimizely setForcedVariation:kExperimentNotRunningKey
```

Comment:
@oakbani If experiment not running, setforcedvariation should return false. right? @yasirfolio3

Reply:
Right now, in PHP/Python SDKs we set forcedvariation if experiment key is a valid experiment irrespective of it's status.